pkg/model/resource/utils.go

```diff
@@ -0,0 +1,51 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resource
+
+import (
+	"path"
+	"strings"
+
+	"github.com/gobuffalo/flect"
+)
+
+// safeImport returns a cleaned version of the provided string that can be used for imports
+func safeImport(unsafe string) string {
+	safe := unsafe
+
+	// Remove dashes and dots
+	safe = strings.Replace(safe, "-", "", -1)
+	safe = strings.Replace(safe, ".", "", -1)
+
+	return safe
+}
+
+// LocalPath returns the default path
+func LocalPath(repo, group, version string, multiGroup bool) string {
+	if multiGroup {
+		if group != "" {
+			return path.Join(repo, "apis", group, version)
+		}
+		return path.Join(repo, "apis", version)
+	}
+	return path.Join(repo, "api", version)
+}
```

Comment:
```suggestion
// DefaultPath returns the default path for the group version informed for this specific project config. 
func (c Config) DefaultPath(group, version string) string {
	if c.MultiGroup {
		if group != "" {
			return path.Join(c.Repo, "apis", group, version)
		}
		return path.Join(c.Repo, "apis", version)
	}
	return path.Join(c.Repo, "api", version)
}
```

Reply:
And then move to the config. Why? Who can answer the question `what is the path for the group X and version Y for this configuration?`

Reply:
The local path, despite requiring to know the repo and wether it is multigroup or not, is not a property of `Config`, and thus, makes no sense semantically to make it an exported method from our `Config` model.

Reply:
I could indeed accept as valid as well as the following. However, I do not like it in the utils approach. Could we change? Also, could we please use `DefaultPath` to return the `Default Path` instead of creating a new nomenclature such as LocalPath to return the default path? 

```
// DefaultPath returns the default path for the group version informed for this specific project config. 
func (r Resource) DefaultPath(c config.Config) string {
	if c.MultiGroup {
		if r.Group != "" {
			return path.Join(c.Repo, "apis", r.Group, r.Version)
		}
		return path.Join(c.Repo, "apis", r.Version)
	}
	return path.Join(c.Repo, "api", r.Version)
}
```

Reply:
`LocalPath` is being used in `NewResource` before you have a resource from whom to call the method.